File: backend/olympic_it_project/src/main/java/org/example/olympic_ot_project/dto/academicyear/UpdateAcademicYearRequest.java
package org.example.olympic_ot_project.dto.academicyear;

import jakarta.validation.constraints.NotBlank;
import lombok.Data;

@Data
public class UpdateAcademicYearRequest {
    @NotBlank(message = "Tên khóa không được để trống")
    private String academicYearName;
}

File: frontend/olympic_it_project/olympic_it_project/lib/dto/admin_manager/academic_year/update_academic_year_request.dart
class UpdateAcademicYearRequest {
  final String academicYearName;

  UpdateAcademicYearRequest({
    required this.academicYearName
  });

  Map<String, dynamic> toJson() {
    return {
      "academicYearName" : academicYearName
    };
  }
}